package.json

```diff
@@ -3,32 +3,42 @@
   "version": "2.0.2-0",
   "description": "An opinionated, isomorphic HTTP client.",
   "license": "BSD-3-Clause",
-  "main": "index.js",
+  "type": "module",
+  "main": "./dist/cjs/index.cjs",
+  "exports": {
+    "require": "./dist/cjs/index.cjs",
+    "import": "./lib/index.js"
+  },
```

Comment:
Since `main` is also here ... we need to test that the non-cjs version is used in some other module test suites and some top-level applications when using `import` syntax and that the cjs version is used when using `require` syntax. We should do that before merging since we're not doing a major version bump here.

Reply:
I've made a simple project here to test that https://github.com/JSAssassin/db-http-client-testing. After updating ky and ky-universal to the latest versions(that uses pure esm), require('@digitalbazaar/http-client') is no longer working and is failing with this error - https://gist.github.com/JSAssassin/2a4a9f26e2dddfdd2cf0f6bb910de125. Since pure esm libs cannot be required. @davidlehn  is currently looking into finding a solution for that problem.

Reply:
@dlongley Due to update ky packages this is a major bump.  We're you thinking we wouldn't update ky* and make it not major?  Those need updating.

Reply:
But, I see that we have to drop support for node 10 and 12. Well, really 12 is the only one that matters there, but ok, I guess we have to major bump for that reason.